src/components/ArticleComments.jsx

```diff
@@ -1,3 +1,49 @@
+import { useEffect,useState } from "react"
+import { getArticleById, getCommentsByArticleId } from "../API"
+import { useParams, Link } from "react-router-dom";
+import { CommentCard } from "./CommentCard";
+import { PostComment } from "./PostComment";
+
 export const ArticleComments = () => {
-    return <h1>Article Comments!</h1>
+
+    const [comments, setComments] = useState([])
+    const [article, setArticle] = useState({})
+    const [loading, setLoading] = useState(true);
+    
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        getCommentsByArticleId(article_id)
+        .then((result) => {
+            setComments(result)
+            setLoading(false)
+        })
+    }, [])
+
+    useEffect(() => {
+        getArticleById(article_id)
+        .then((response) => {
+          setArticle(response)
+        })
+        .catch((err)=> {
+            console.log(err, '<<<<')
+        })
+    }, []);
+
+    if (loading) {
+        return <div>Loading!</div>;
+    }
+
+
+    return <div>
+        <h2>{article.title}</h2>
+        <h2>Comments</h2>
+        <PostComment setComments={setComments} article_id={article_id}/>
```

Comment:
There is a post comment component here but there is currently no comment posting functionality on your app?